msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -167,6 +167,9 @@ The `{hostName}` [URL placeholder value](./access-teams-context.md#get-context-b
 * **Do** use *hostName* to differentiate the theme of your application based on the host it's running in. For example, you can use Microsoft Teams purple as the main accent color when running in Teams, and Outlook blue when running in Outlook.
 * **Do** use *hostName* to differentiate messages shown to the user based on which host it's running in. For example, show *Manage your tasks in Microsoft 365* when running in Microsoft 365 on the web, and *Manage your tasks in Teams* when running in Teams.
 
+> [!TIP]
+> Best practice is to target the runtime requirements and dependencies of your app in a host-agnostic way, rather than special casing your app code with host-specific logic. For more info, see [Specify Microsoft 365 host runtime requirements in your app manifest](../../m365-apps/specify-runtime-requirements.md).
```

Comment:
```suggestion
> The best practice is to target the runtime requirements and dependencies of your app in a host-agnostic way, rather than special casing your app code with host-specific logic. For more info, see how to [specify Microsoft 365 host runtime requirements in your app manifest](../../m365-apps/specify-runtime-requirements.md).
```